apps/portal-front/src/components/trials/trials-tab.tsx

```diff
@@ -0,0 +1,371 @@
+              cell: ({ row }: { row: { original: trials_fragment$data } }) => {
+                return (
+                  <span className="truncate">
+                    {row.original.end_date
+                      ? formatDate(row.original.end_date, 'DATE_FULL')
+                      : '-'}
+                  </span>
+                );
+              },
+            },
+          ]),
+      ...(type === TrialsTabType.Running
+        ? [
+            {
+              accessorKey: 'remainingDays',
+              id: 'remainingDays',
+              header: t('TrialsDashboard.Columns.RemainingDays'),
+              enableSorting: false,
+              cell: ({ row }: { row: { original: trials_fragment$data } }) => {
+                if (!row.original?.end_date) return <>-</>;
+                const target = new Date(row.original.end_date);
+                const diffInDays = daysUntil(target);
+                return <span className="truncate">{diffInDays}</span>;
+              },
+            },
+          ]
+        : []),
+      {
+        accessorKey: 'hub_status',
+        id: 'hub_status',
+        header: t('TrialsDashboard.Columns.Status'),
+      },
+      {
+        accessorKey: 'region',
+        id: 'region',
+        header: t('TrialsDashboard.Columns.Region'),
+      },
+      ...(type === TrialsTabType.Cancelled
+        ? [
+            {
+              header: t('TrialsDashboard.Columns.CancellationDate'),
+            },
+            {
+              header: t('TrialsDashboard.Columns.CancellationOwner'),
+            },
+          ]
+        : []),
+      ...(type === TrialsTabType.Running || type === TrialsTabType.Waiting
+        ? [
+            {
+              accessorKey: 'actions',
+              id: 'actions',
+              enableHiding: false,
+              enableSorting: false,
+              enableResizing: false,
+              header: undefined,
+              cell: () => {
+                return (
+                  <>
+                    {(type === TrialsTabType.Running ||
+                      type === TrialsTabType.Waiting) && (
+                      <Button
+                        variant="ghost-destructive"
+                        size="icon"
+                        className="border m-1">
+                        <CloseIcon className="h-4 w-4" />
+                      </Button>
+                    )}
+                    {type === TrialsTabType.Waiting && (
+                      <Button
+                        variant="ghost-primary"
+                        size="icon"
+                        className="border m-1">
+                        <ArrowUpwardIcon className="h-4 w-4" />
+                      </Button>
+                    )}
+                  </>
+                );
+              },
+            },
+          ]
+        : []),
+    ],
+    [t, type]
+  );
+
+  const queryData = useLazyLoadQuery<trialsListQuery>(TrialsListQuery, {
+    count: 100,
```

Comment:
You can load initial query with `pageSize`. You will have to set `pageSize` to 100 in `trial-list-localstorage`